src/manage_sql_report_formats.c

```diff
@@ -2406,6 +2407,28 @@ report_format_param_type_min (report_format_t report_format, const char *name)
   return min;
 }
 
+static int
+report_format_param_value_in_options (report_format_param_t param,
+                                      const char *value)
```

Comment:
You may want to add a description for this function.

Reply:
I've added one in ae41309